try-catch-finally-lib/main.rkt

```diff
@@ -34,7 +34,7 @@
   [(_ b:body ...+ c:catch-clause ...)
    #'(with-handlers ([c.pred c.handler] ...) b ...)]
   [(_ b:body ...+ c:catch-clause ... f:finally-clause)
-   #'(call-with-try-funally
+   #'(call-with-try-finally
```

Comment:
Oh, oops. Thank you

Reply:
It's good practice when you fix something to add a test that would-have-failed before your fix, that shows the fix succeeded. So I added a test here: https://github.com/AlexKnauth/try-catch-finally/commit/ae71f04fe3ef793ed9338a7ee662f72ccc32222b